src/_shared/scss/_core.scss

```diff
@@ -112,7 +112,7 @@ a.u-faux-block-link__overlay {
 .button {
     border: 0;
     border-radius: 1000px;
-    font: bold 13px/20px $f-sans-serif-text;
+    font: bold 13px/0px $f-sans-serif-text;
```

Comment:
Does this not make the font-size 0?

Reply:
It makes the line height 0px - I did that to vertical-align the button text in the centre (the studio said the text was too high and mis-aligned with the arrow), sure there's a better way of doing that.

Reply:
Hmm, what is the content of the button? If there isn't any text then presumably we can just set `line-height: 0`; maybe we can look at this together and see if there is a cleaner way of achieving this (if `line-height` doesn't work)

Reply:
I just checked and we can just delete the line-height bit altogether I think and that'll do the same thing. The button small div is:

![image](https://user-images.githubusercontent.com/4101937/35047755-c90866c8-fb92-11e7-86b3-d4c889640611.png)

Reply:
Nice one, deleting code is always a bonus 😃